File: Ecole--Edtech1/src/GestionBundle/Entity/Permutation.php
<?php

namespace GestionBundle\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;
/**
 * Permutation
 *
 * @ORM\Table(name="permutation")
 * @ORM\Entity(repositoryClass="GestionBundle\Repository\PermutationRepository")
 */
class Permutation
{
    /**
     * @var int
     *
     * @ORM\Column(name="id", type="integer")
     * @ORM\Id
     * @ORM\GeneratedValue(strategy="AUTO")
     */
    private $id;



    /**
     * @var string
     *
     * @ORM\Column(name="classe_s", type="string", length=255)
     * @Assert\Length(max=3)
     */
    private $classeS;

    /**
     * @var string
     *
     * @ORM\Column(name="raison", type="string", length=255)
     */
    private $raison;

    /**
     * @var string
     *
     * @ORM\Column(name="etat", type="string", length=255)
     */
    private $etat;

    /**
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User", inversedBy="noteseleve")
     * @ORM\JoinColumn(name="eleve_id",referencedColumnName="id")
     */
    private $eleve;

    /**
     * @ORM\ManyToOne(targetEntity="AppBundle\Entity\User")
     * @ORM\JoinColumn(name="parent",referencedColumnName="id")
     */
    private $parent;

    /**
     * @return mixed
     */


    /**
     * @var \DateTime
     *
     * @ORM\Column(name="date", type="datetime")
     */
    private $date;

    /**
     * @return int
     */
    public function getId()
    {
        return $this->id;
    }

    /**
     * @param int $id
     */
    public function setId($id)
    {
        $this->id = $id;
    }

    /**
     * @return string
     */
    public function getClasseS()
    {
        return $this->classeS;
    }

    /**
     * @param string $classeS
     */
    public function setClasseS($classeS)
    {
        $this->classeS = $classeS;
    }

    /**
     * @return string
     */
    public function getRaison()
    {
        return $this->raison;
    }

    /**
     * @param string $raison
     */
    public function setRaison($raison)
    {
        $this->raison = $raison;
    }

    /**
     * @return string
     */
    public function getEtat()
    {
        return $this->etat;
    }

    /**
     * @param string $etat
     */
    public function setEtat($etat)
    {
        $this->etat = $etat;
    }

    /**
     * @return mixed
     */
    public function getEleve()
    {
        return $this->eleve;
    }

    /**
     * @param mixed $eleve
     */
    public function setEleve($eleve)
    {
        $this->eleve = $eleve;
    }

    /**
     * @return mixed
     */
    public function getParent()
    {
        return $this->parent;
    }

    /**
     * @param mixed $parent
     */
    public function setParent($parent)
    {
        $this->parent = $parent;
    }

    /**
     * @return \DateTime
     */
    public function getDate()
    {
        return $this->date;
    }

    /**
     * @param \DateTime $date
     */
    public function setDate($date)
    {
        $this->date = $date;
    }



}

File: ProjetPi/src/com/mycompany/myapp/entities/Permutation.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.mycompany.myapp.entities;

/**
 *
 * @author Selim Chikh Zaouali
 */
public class Permutation {

    private int id;
    private String classe_s;
    private String raison;
    private String date;
    private String etat;
    private int eleve_id;
    private int parent;
    private String enfant;

    public Permutation() {

    }

    public Permutation(String classe_s, String raison, String date, String etat, int eleve_id, int parent) {
        this.classe_s = classe_s;
        this.raison = raison;
        this.date = date;
        this.etat = etat;
        this.eleve_id = eleve_id;
        this.parent = parent;
    }

    public Permutation(String classe_s, String raison, String date, String etat, int eleve_id, int parent, String enfant) {
        this.classe_s = classe_s;
        this.raison = raison;
        this.date = date;
        this.etat = etat;
        this.eleve_id = eleve_id;
        this.parent = parent;
        this.enfant = enfant;
    }

    public Permutation(int id, String classe_s, String raison, String date, String etat, int eleve_id, int parent, String enfant) {
        this.classe_s = classe_s;
        this.raison = raison;
        this.date = date;
        this.etat = etat;
        this.eleve_id = eleve_id;
        this.parent = parent;
        this.enfant = enfant;
        this.id = id;
    }

    public void setEnfant(String enfant) {
        this.enfant = enfant;
    }

    public String getEnfant() {
        return enfant;
    }

    public void setId(int id) {
        this.id = id;
    }

    public void setClasse_s(String classe_s) {
        this.classe_s = classe_s;
    }

    public void setRaison(String raison) {
        this.raison = raison;
    }

    public void setDate(String date) {
        this.date = date;
    }

    public void setEtat(String etat) {
        this.etat = etat;
    }

    public void setEleve_id(int eleve_id) {
        this.eleve_id = eleve_id;
    }

    public void setParent(int parent) {
        this.parent = parent;
    }

    public int getId() {
        return id;
    }

    public String getClasse_s() {
        return classe_s;
    }

    public String getRaison() {
        return raison;
    }

    public String getDate() {
        return date;
    }

    public String getEtat() {
        return etat;
    }

    public int getEleve_id() {
        return eleve_id;
    }

    public int getParent() {
        return parent;
    }

}
